exercises/word-count/test/word_count_test.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:test/test.dart';
+import 'package:word_count/word_count.dart';
+
+final WordCount myWordCounter = new WordCount();
+
+void main() {
+  group("WordCount: Simple Tests - ", simpleTests);
+  group("WordCount: Ignore special characters - ", ignoreSpecialCharacters);
+  group("WordCount: Works with numbers too - ", notJustWords);
+  group("WordCount: Edge case - ", edgeCases);
+}
+
+void simpleTests() {
+  test("count one word", () {
+    expect(myWordCounter.countWords("word"), equals({"word": 1}));
+  });
```

Comment:
Whatever is the first test as defined in the canonical.json, should have have `}, skip: false);` at the end, all other tests should have `}, skip: true);`